tests/python/examples/simulation/test_mpm99.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+
+import taichi as ti
+
+FRAMES = 100
+
+
+def test_mpm99():
+    from taichi.examples.simulation.mpm99 import dt, initialize, substep
+
+    initialize()
+    for i in range(FRAMES):
+        for s in range(int(2e-3 // dt)):
+            substep()
+
+
+def video_mpm99(result_dir):
+    from taichi.examples.simulation.mpm99 import (dt, initialize, material,
+                                                  substep, x)
+
+    video_manager = ti.VideoManager(output_dir=result_dir,
```

Comment:
nit: Do we really want to keep `VideoManager` in the top level in the future...?

Reply:
This is still under discussion. My personal idea is no :-)